src/abstractdataframe/abstractdataframe.jl

```diff
@@ -871,15 +871,22 @@ function nonunique(df::AbstractDataFrame)
     gslots = row_group_slots(ntuple(i -> df[i], ncol(df)), Val(true))[3]
     # unique rows are the first encountered group representatives,
     # nonunique are everything else
-    res = fill(true, nrow(df))
+    res = trues(nrow(df))
```

Comment:
Isn't that a bit slower? There may be a speed-memory tradeoff, and in general we tend to favor speed since we assume the data frame stores many columns so it doesn't really matter to allocate another one temporarily.